Build failure on Swift 5.4

`swiftplot` fails to build on Swift 5.4 (Ubuntu 20.04):

```
$ swift build
:
:

.
:
:

:
:

.../swiftplot/Sources/AGGRenderer/CPPAGGRenderer/CPPAGGRenderer.cpp:6:10: fatal error: could not build module 'AGG'
#include "agg_basics.h"
 ~~~~~~~~^~~~~~~~~~~~~~
.
[38/73] Compiling AGG agg_font_freetype.cpp
```

--- a/Sources/AGG/include/agg_blur.h
+++ b/Sources/AGG/include/agg_blur.h
@@ -32,6 +32,7 @@
 #include "agg_array.h"
 #include "agg_pixfmt_base.h"
 #include "agg_pixfmt_transposer.h"
+#include "agg_renderer_base.h"
 
 namespace agg
 {

